common/transport/amqp/src/sender_link.ts

```diff
@@ -315,7 +316,7 @@ export class SenderLink extends EventEmitter implements AmqpLink {
             this._indicatedError = undefined;
             this._senderCloseOccurred = true;
             if (error) {
-              debugErrors(this.toString() + ': In sender attached state - close event for ' + context.sender.name + ' already indicated error is: ' + this._getErrorName(error));
+              debugErrors(this.toString() + ': In sender attached state - close event for ' + context.sender.name + ' already indicated error is: ' + getErrorName(error));
```

Comment:
_very minor_ nit: Template literals would be more readable and slightly shorter than concatenating strings, i.e.,
```suggestion
              debugErrors(`${this.toString()}: In sender attached state - close event for ${context.sender.name} already indicated error is: ${getErrorName(error)}`);
```